Makefile

```diff
@@ -9,10 +9,10 @@ lint:
 	flake8
 
 test: all
-	nosetests
+	py.test
```

Comment:
`py.test` is required for the plugin integration tests added in this PR (`nosetests` explodes on `with redirected_stdio(stdout=os.devnull):`).